### DISCUSSION TITLE
Scrambled UI components after fresh installation of 24.0.4

### DISCUSSION THREAD

**Discussion by Calab** [dbeaver/dbeaver#32920] [Q&A]
I just downloaded and install DBeaver 24.0.4 onto my Windows 11 PC. When I open the program, parts of the UI are "scrambled". How can I correct this?

![image](https://github.com/dbeaver/dbeaver/assets/316725/fde269f2-5e45-433f-ab2e-03fd3099aa36)

**Comment by E1izabeth** [comment]
It's a known issue https://github.com/dbeaver/dbeaver/issues/9455.
You could try changing `Main font` in settings. Sometimes it helps.
![image](https://github.com/dbeaver/dbeaver/assets/28875055/9330a74e-8989-4c16-b878-e4a12a8af0eb)

![image](https://github.com/dbeaver/dbeaver/assets/28875055/c7d0e740-6e2d-488f-9889-da2d68d2575a)
